shell.nix

```diff
@@ -0,0 +1,23 @@
+with import <nixpkgs> { };
+
+let
+  moz_overlay = import (builtins.fetchTarball
+    "https://github.com/mozilla/nixpkgs-mozilla/archive/master.tar.gz");
+  nixpkgs = import <nixpkgs> { overlays = [ moz_overlay ]; };
```

Comment:
Should it be pinned? You know that a lot better than I do. If it's not pinned and the user also has another project, it will result in saving space.. So i'm not sure

Reply:
it should be pinned, in pure evaluation when fetching from the internet, it should always have a sha256 hash

Reply:
The issue is that elkowar often updates his nightly. And this particular version is already out of date (A expermintal feature doesn't need a flag anymore, in the latest version, but it does in the pinned one). 
And it saves spaces if it's the latest, as other projects are more likely to have the same.

Reply:
ok, I have set the rev to point at the latest master, and updated the other review to also use the latest rust nightly